app/routes/circulars/circulars.server.ts

```diff
@@ -69,11 +82,27 @@ export async function search({
   } = await client.search({
     index: 'circulars',
     body: {
-      query: query && {
-        multi_match: { query, fields: ['submitter', 'subject', 'body'] },
+      query: {
+        bool: {
+          must: query
+            ? {
+                multi_match: {
+                  query,
+                  fields: ['submitter', 'subject', 'body'],
+                },
+              }
+            : undefined,
+          filter: {
+            range: {
+              createdOn: {
+                gte: startTime,
+                lte: endTime,
+              },
+            },
+          },
+        },
       },
-      fields: ['subject'],
-      _source: false,
+      _source: ['subject', 'createdOn'],
```

Comment:
This was probably a typo. I think that the `_source` argument is supposed to be a boolean.

Reply:
Looking through some docs ([1](https://www.elastic.co/guide/en/elasticsearch/reference/current/mapping-source-field.html)) ([2](https://opster.com/guides/elasticsearch/glossary/elasticsearch-source/)), I think I implemented the syntax slightly incorrectly, but setting it to false seems to break the circulars indexing on initial load. I probably need to look this over a bit more to understand what's happening, but altering it to the following seems to work (doesn't fix the query submission issue that I'm encountering though):

```
      _source: {
        includes: ['subject', 'createdOn'],
      }
```